chart/templates/cdn-invalidator-sub/deployment.yaml

```diff
@@ -40,7 +40,7 @@ spec:
         {{- include "common.tplvalues.render" (dict "value" .Values.cdnInvalidatorSub.podLabels "context" $) | nindent 8 }}
         {{- end }}
     spec:
-      serviceAccountName: {{ template "carto.cdnInvalidatorSub.serviceAccountName" . }}
+      serviceAccountName: {{ template "carto.commonGcpApi.serviceAccountName" . }}
```

Comment:
I was thinking a lot regarding if that was the right way to proceed (delete the ability to customize the SA per component) and... I don't have a clear decision because if we leave the original way, we should edit all the templates so it's not easy to debug when a component uses a SA or another reading the deployment yaml but the old way allow us to attach a custom SA if required.

But... I think that, if we add the same freedom to configure the common SA as we already had with the components, we can keep this change.